### PR TITLE
chore(scan): remove most references to HMPB templates

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview

Refs #3374, follow-up to #3381.

## Demo Video or Screenshot
n/a

## Testing Plan
Automated.

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/central-scan/backend/src/interpreter.ts [hunk 1]
```diff
@@ -267,38 +208,6 @@ export class Interpreter {
...
-        return hmpbResult;
-      }
-    } catch (error) {
-      assert(error instanceof Error);
-      debug('interpretHMPBFile failed: %s', error.message);
-    }
-
     try {
       debug(
         'assuming ballot is a HMPB that could not be interpreted (QR data: %s)',
```
Comment (jonahkagan) [inline apps/central-scan/backend/src/interpreter.ts:213]: Do we want to keep these cases for invalid HMPBs around?

File: libs/ballot-interpreter-vx/jest.config.js [hunk 2]
```diff
@@ -5,13 +5,13 @@ const shared = require('../../jest.config.shared');
...
   coverageThreshold: {
     global: {
-      statements: 91,
-      branches: 77,
-      lines: 92,
-      functions: 85,
+      statements: 0,
+      branches: 0,
+      lines: 0,
+      functions: 0,
```
Comment (jonahkagan) [inline libs/ballot-interpreter-vx/jest.config.js:14]: Are you leaving this at 0? If so, can you add a comment explaining why?

**Review by jonahkagan — APPROVED** [review]

File: apps/central-scan/backend/src/interpreter.ts (hunk 1 above)
Comment (eventualbuddha) [reply apps/central-scan/backend/src/interpreter.ts:213]: I'm removing them in a later PR.

File: libs/ballot-interpreter-vx/jest.config.js (hunk 2 above)
Comment (eventualbuddha) [reply libs/ballot-interpreter-vx/jest.config.js:14]: I reset them in a later PR after changing the purpose of the library to be for BMD ballots.

**Pull request merged**